test/Player.spec.ts

```diff
@@ -39,46 +37,67 @@ describe("Player", () => {
         new TestGame(config);
     });
 
+    beforeEach(() => {
+        scene.onTestStart((_player: Player, _block: Block) => {
+            player = _player;
+            block = _block;
+        });
+    });
+
     it("should bounce from ground", (done) => {
         var playerGroundedPos = block.getBounds().top - player.getBounds().height / 2;
-        assert(player.y < playerGroundedPos);
+        expect(player.y).to.be.lessThan(playerGroundedPos);
```

Comment:
Just leaving a note, the coordinate system in Phaser has y going downward the higher the y-coord value. Meaning that to check for something above another thing, you check if its y is less than the other thing's y. Kinda confusing when writing these tests IMO because I'm expecting to check for greater than, like a traditional Cartesian coordinate system.